source/index.html

```diff
@@ -41,6 +41,8 @@
 	<link rel="stylesheet" href="documentation.css">
 </head>
 <body>
+
 <div id="ide-container" style="position: fixed; left: 0; top: 0; width: 100vw; height: 100vh;"></div>
+</div>
```

Comment:
The additional closing tag is unneeded. The div is already closed in the previous line.

```suggestion
<div id="ide-container" style="position: fixed; left: 0; top: 0; width: 100vw; height: 100vh;"></div>
```